project/index.mjs

```diff
@@ -0,0 +1,173 @@
+/** Route handler for the /project endpoint */
+import express from 'express'
+import * as logic from './project.mjs'
+import * as utils from '../utilities/shared.mjs'
+import cors from 'cors'
+
+let router = express.Router()
+router.use(
+  cors({
+    methods: 'GET',
+    allowedHeaders: [
+      'Content-Type',
+      'Content-Length',
+      'Allow',
+      'Authorization',
+      'Location',
+      'ETag',
+      'Connection',
+      'Keep-Alive',
+      'Date',
+      'Cache-Control',
+      'Last-Modified',
+      'Link',
+      'X-HTTP-Method-Override',
+    ],
+    exposedHeaders: '*',
+    origin: '*',
+    maxAge: '600',
+  })
+)
+
+// Send a successful response with the appropriate JSON or alternate response based on params
+export function respondWithProject(req, res, project) {
+  const id = project['@id'] ?? project.id ?? null
+
+  let textType = req.query.text
+  let image = req.query.image
+  let lookup = req.query.lookup
+  let view = req.query.view
+  let responseType = [textType, image, lookup, view]
+    .find(elem => elem !== undefined) // Only one response type is valid
```

Comment:
You will actually want to throw an error here if more than one of this is !==undefined.